projects/admin/src/app/record/custom-editor/libraries/library.component.ts

```diff
@@ -25,17 +25,23 @@ import { ToastrService } from 'ngx-toastr';
 import { UserService } from '@rero/shared';
 import { Library } from '../../../class/library';
 import { LibraryFormService } from './library-form.service';
+import { Subscription } from 'rxjs';
 
 @Component({
   selector: 'admin-libraries-library',
   templateUrl: './library.component.html',
   styleUrls: ['./library.component.scss']
 })
-export class LibraryComponent implements OnInit {
+export class LibraryComponent implements OnInit, OnDestroy {
 
   public library: Library;
   public libForm: FormGroup;
-  public organisationPid;
+  public organisationPid: string;
+
+  /** List of internal notification types */
+  internalTypes = ['booking', 'request', 'transit_notice'];
```

Comment:
Should be set in a Enum class into `class/library.ts` file. As ItemStatus, or NoteType